Code/Source/RobotImporter/URDF/URDFPrefabMaker.cpp

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "RobotImporter/URDF/URDFPrefabMaker.h"
+#include "Frame/ROS2FrameComponent.h"
+#include "RobotControl/ROS2RobotControlComponent.h"
+#include "RobotImporter/URDF/CollidersMaker.h"
+#include "RobotImporter/URDF/PrefabMakerUtils.h"
+
+#include "RobotImporter/RobotImporterWidget.h"
+
+#include <AzCore/IO/FileIO.h>
+#include <AzCore/Utils/Utils.h>
+#include <AzToolsFramework/API/EditorAssetSystemAPI.h>
+#include <AzToolsFramework/Entity/EditorEntityHelpers.h>
+#include <PhysX/MeshAsset.h>
+
+#include <regex> // TODO - we are currently replacing package:// with an absolute path
```

Comment:
This regex include isn't used(Also there is an AzCore/std/string/regex.h implementation as well)